bin/python_scripts/solr_reindex_package_ids.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Set the FILENAME_BASE to the txt file containing the list of package_ids
+#
+# Execute script like this -
+# python solr_reindex_package_ids.py
+#
+
+import os
+import sys
+import subprocess
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+FILENAME_BASE = "reindex_datasets-package_ids_28-31_07"
```

Comment:
Are you intending to commit this as a file? I notice line 37 seems to open "reindex_datasets-package_ids_28-31_07.txt"

Reply:
not really, I'll change it to some generic filename